interfaces/apparmor/template_vars.go

```diff
@@ -23,15 +23,18 @@ import (
 	"bytes"
 	"fmt"
 
+	"github.com/snapcore/snapd/interfaces/dbus"
 	"github.com/snapcore/snapd/snap"
 )
 
 // templateVariables returns text defining apparmor variables that can be used in the
 // apparmor template and by apparmor snippets.
-func templateVariables(info *snap.Info) []byte {
+func templateVariables(info *snap.Info, securityTag string) []byte {
 	var buf bytes.Buffer
 	fmt.Fprintf(&buf, "@{SNAP_NAME}=\"%s\"\n", info.Name())
 	fmt.Fprintf(&buf, "@{SNAP_REVISION}=\"%s\"\n", info.Revision)
+	fmt.Fprintf(&buf, "@{SNAP_LABEL_DBUS}=\"%s\"\n",
```

Comment:
@zyga, I recommended it we bring it back. It was one of variables that was removed because nothing used it yet. We agreed then that we'd reintroduce it when Personal came along, since trusted helpers on Personal will use the DBusified security label as part of their DBus API.

That said, SNAP_LABEL_DBUS sounds a little weird. I know why it is named this way: it is the full security label of the process as a dbus string. AppArmor itself understands the '@{profile}' variable and expands it to the security label of the process. I think '@{PROFILE_DBUS} might be better in that light. What do others think?